FordLabs/PeopleMover
# Cypress in GitHub Actions and Git commit

## Issue
Connects #272 

## What was done
- [x] Run cypress tests on commit
- [x] Run backend tests on commit
- [x] Run Cypress tests in GitHub Actions
- [x] Move all actions into one workflow
- [x] Save cypress info in artifacts for debugging purposes (videos AND screenshots)

## How to test
1. Run git commit and make sure cypress and backend tests run
2. Push a branch and make sure cypress runs
